targets/TARGET_STM/TARGET_STM32F0/serial_device.c

```diff
@@ -742,13 +742,12 @@ void serial_set_flow_control(serial_t *obj, FlowControl type, PinName rxflow, Pi
 {
     struct serial_s *obj_s = SERIAL_S(obj);
 
-    // Determine the UART to use (UART_1, UART_2, ...)
+    // Checked used UART name (UART_1, UART_2, ...)
     UARTName uart_rts = (UARTName)pinmap_peripheral(rxflow, PinMap_UART_RTS);
-    UARTName uart_cts = (UARTName)pinmap_peripheral(txflow, PinMap_UART_CTS);
+    MBED_ASSERT((UARTName)pinmap_merge(uart_rts, obj_s->uart) != (UARTName)NC);
```

Comment:
why is this assert ? What will happen in the release build if asserts is not active, wont be merged.

Reply:
Good catch.
I have updated the code